rmgpy/rmg/reactors.py

```diff
@@ -35,25 +35,10 @@
 import logging
 import itertools
 
-if __debug__:
-    try:
-        from os.path import dirname, abspath, join, exists
-
-        path_rms = dirname(dirname(dirname(abspath(__file__))))
-        from julia.api import Julia
-
-        jl = Julia(sysimage=join(path_rms, "rms.so")) if exists(join(path_rms, "rms.so")) else Julia(compiled_modules=False)
-        from pyrms import rms
-        from diffeqpy import de
-        from julia import Main
-    except Exception as e:
-        import warnings
-
-        warnings.warn("Unable to import Julia dependencies, original error: " + str(e), RuntimeWarning)
-else:
-    from pyrms import rms
-    from diffeqpy import de
-    from julia import Main
+from diffeqpy import de
```

Comment:
I don't think we actually really need diffeqpy here, we should be able to get CVODE_BDF from juliacall/Sundials instead. We should be able to do the same in pdep/sls.py except I think with SciMLBase(?) and then we should be able to remove diffeqpy as an RMG dependency.